index.html

```diff
@@ -4,6 +4,8 @@
   <link type="text/css" rel="stylesheet" href="style.css" />
 </head>
 <body>
+  <div id="app">
+  </div>
```

Comment:
Since there is no content, you might consider moving this div to the line of the opening tag.